adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBBannerRenderer.m

```diff
@@ -117,14 +111,10 @@ - (PAGBannerAdSize)bannerSizeFormGADAdSize:(GADAdSize)gadAdSize error:(NSError *
   } else if (size.height == kPAGBannerSize300x250.size.height) {
     return kPAGBannerSize300x250;
   }
-
-  if (error) {
-    *error = GADMAdapterPangleErrorWithCodeAndDescription(
-        GADPangleErrorBannerSizeMismatch,
-        [NSString stringWithFormat:@"Invalid size for Pangle mediation adapter. Size: %@",
-                                   NSStringFromGADAdSize(gadAdSize)]);
+  if (gadAdCGSize.width >= kPAGBannerSize728x90.size.width) {
+    return kPAGBannerSize728x90;
   }
-  return (PAGBannerAdSize){CGSizeZero};
```

Comment:
Revert this change. If the publisher requests an invalid banner ad size, we should always return an error.